src/bytes.ts

```diff
@@ -0,0 +1,30 @@
+import { Atom, bigintToDataView, cue_bytes, Noun } from '@urbit/nockjs'
+
```

Comment:
Changes in this file are awesome — js buffers are much easier to reason about in TS is seems — thanks for killing this ugly dep.